affinity/cpp-20/d0796r2.md

```diff
@@ -326,9 +327,9 @@ If a particular policy or algorithm requires to access placement information, th
      public:
 
       using native_affinity_type = see-below;
-      using error_type = see-below
+      using error_type = see-below;
 
-      affinity_query(execution_resource &&, execution_resource &&) noexcept;
+      affinity_query(execution_resource & const , std::pmr::memory_resource & const ) noexcept;
```

Comment:
#49 suggests we should get rid of pmr::memory_resource, should we use simply memory_resource here? Or should we do a separate pull request for that change?

Reply:
Let's do a follow up discussion on #49 and follow up PR.

Reply:
Makes sense, lets keep your change until #49 is sorted out.

Reply:
I agree. We decided that for the purposes of affinity-based allocation it didn't make sense to use the pmr memory resource, however after separating the execution and memory topologies, I think the pmr memory resource could be a good choice to represent a node within the memory topology, even if that is then passed to a concrete affinity aware allocator type. Though I think we should (as @hcedwar suggested in https://github.com/codeplaysoftware/standards-proposals/issues/41) make it a type which is derived from the pmr memory resource so that it can be made iterable. I think we need to discuss this further.